setup.py

```diff
@@ -0,0 +1,4 @@
+from sugar3.activity import bundlebuilder
+
+if __name__ == "__main__":
+    bundlebuilder.start()
```

Comment:
Please add missing newline at the end of the file

Reply:
Done